src/main/kotlin/no/nav/syfo/dialogmote/api/domain/DialogmotedeltakerArbeidstakerVarselDTO.kt

```diff
@@ -12,4 +13,7 @@ data class DialogmotedeltakerArbeidstakerVarselDTO(
     val fritekst: String,
     val document: List<DocumentComponentDTO>,
     val brevBestiltTidspunkt: LocalDateTime?,
+    val svarType: DialogmoteSvarType?,
```

Comment:
For frontend hadde det vært kjekt om strukturen i for svar i disse DTOene til Modia var mer lik svar i `DialogmotedeltakerBehandlerVarselDTO`, typ et (optional) svar-objekt med createdAt, svarType og tekst (optional).

Reply:
Ja, skal forsøke å forbedre dette 👍